pipelines/broad/arrays/imputation_beagle/ImputationBeaglePreChunk.wdl

```diff
@@ -113,10 +95,17 @@ workflow ImputationBeagle {
                     output_basename = "imputed_sites"
             }
 
-            if (CheckChunksBeagle.valid) {
+            if (!CheckChunksBeagle.valid) {
+                call utils.ErrorWithMessage as FailedChunk {
```

Comment:
curious how this will work if there's say 10 chunks and 1 fails. I worry that the one will call this ErrorWithMessage task, but the other 9 will go on to impute, and until they fix the issue with nested task failures not failing the other tasks properly, they're all going to complete before Cromwell "catches" that there's a failed task and actually fails the whole workflow. have you tested this scenario? is this worth worrying about? (seems like a lot of compute wasted)